base/debian/cuttlefish-base.cuttlefish-host-resources.init

```diff
@@ -108,31 +108,46 @@ destroy_tap() {
 # Create a mobile tap interface
```

Comment:
it is generic, not only for mobile, isn't it?

Reply:
Yes, it's my mistake. Thanks :)